packages/expo-face-detector/CHANGELOG.md

```diff
@@ -4,13 +4,22 @@
 
 ### 🛠 Breaking changes
 
+- Replace exported `Constants` with String Enums (`FaceDetectorMode`, `FaceDetectorLandmarks` and `FaceDetectorClassifications`). ([#14179](https://github.com/expo/expo/pull/14179) by [@Simek](https://github.com/Simek))
+
+```diff
+- FaceDetector.Constants.Mode.*
++ FaceDetector.FaceDetectorMode.*
+```
+
```

Comment:
We aim to keep CHANGELOG as short as possible with no extra elements (our scripts does not handle additional info well 😅) and according to @tsapeta: CHANGELOGs are not the place to provide migration guides (https://github.com/expo/expo/pull/14061#discussion_r692394118)
```suggestion
```